Code Review: Issue 14: Loan Functionality: Bank has unlimited money

KshipraShetty opened the pull request:
Check if any changes required

KshipraShetty committed:
ISSUE14:migrated and added the seeder for bank

KshipraShetty committed:
Issue14: Basic test for getting the bank amount

KshipraShetty committed:
Issue 14: Added new test case

KshipraShetty committed:
Issue 14: added the get method to get the max value

KshipraShetty committed:
Issue 14: GET:made changes

KshipraShetty committed:
Issue 14: Post added

KshipraShetty committed:
Issue 14: added test for post method

KshipraShetty committed:
Issue 14: restructured

KshipraShetty committed:
Issue 14: added a test case for POST

Ghost---Shadow reviewed (changes requested):
1. The PR is not up-to-date with develop (Run `Use git pull origin develop --rebase`, on your branch) 
2. Test cases are not passing :(

mrsauravsahu committed:
- Updated test script in package.json
- Added .vscode launch and tasks config
- Fixed eslint errors in migration
- Updated server fallback port to 8080
- Fixed failing tests
- Updated path to /api/max-amount
